### ISSUE TITLE
stack-create fails when multiple capabilities are required

### ISSUE THREAD

**Issue by andrewlorien** [bash-my-aws/bash-my-aws#342]
the --capabilities option only finds the first capability.  
`stack-create  pgis-prod-Cloudwatch2S3 cloudformation/CloudWatch2S3.template cloudformation/parameters/prod/prod-CloudWatch2S3.json --capabilities=CAPABILITY_IAM CAPABILITY_AUTO_EXPAND`

pull request coming soon.

**Comment by andrewlorien** [comment]
I agree with https://github.com/bash-my-aws/bash-my-aws/issues/146, but I don't want to break backward compatibility problems.  How about if i just parse the end of the command for all allowed capabilities (with or without the `--capabilities=` ) and include whatever i find?

**Comment by andrewlorien** [comment]
I agree with the suggestion at https://github.com/bash-my-aws/bash-my-aws/issues/146 that the capabilities syntax could be more intuitive.  How about i just parse the end of the commandline for any legal capabilities.  That way it's still backward compatible with anybody using `--capabilities=CAPABILITY_IAM` .

**Comment by mbailey** [comment]
One issue I have with the capabilities IAM is that I forget the syntax I need to use it.

I think the most beautiful behaviour would be to prompt the user with required options if they are omitted and get confirmation before running the updated command

**Comment by mbailey** [comment]
> I agree with the suggestion at https://github.com/bash-my-aws/bash-my-aws/issues/146 that the capabilities syntax could be more intuitive.  How about i just parse the end of the commandline for any legal capabilities.  That way it's still backward compatible with anybody using `--capabilities=CAPABILITY_IAM` .

I think that sounds like a good solution and would be easy to add to shell completion.

Perhaps we could pause all arguments (rather than just at end) for any capabilities, and enable them if they are present?

**Comment by andrewlorien** [comment]
> Side note: One issue I have with the capabilities IAM is that I forget the syntax I need to use it.
> 
> I think the most beautiful behaviour would be to prompt the user with required options if they are omitted and get confirmation before running the updated command

Any example of a confirmation-with-details already in bash-my-aws?  I'd like to be consistent.
